app/models/case/overturned_ico/base.rb

```diff
@@ -73,6 +73,10 @@ def link_related_cases
     [original_case, original_ico_appeal].each { |source_case| link_cases_related_to(source_case) }
   end
 
+  def identifier
+    message
+  end
```

Comment:
Tests?

Reply:
It is only used in tests.  We could make it private, in which case we'd have to use __send__ in the tests. - Shall I do that?

Reply:
How difficult will it be to make it private and then to update the test to use **send**.

Perheps add a comment above stating that it is only for testing etc. so the next dev to look at it will understand what its for and why we are using it.